python/py-glcontext/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-glcontext
+version             2.3.5
+revision            0
+
+license             MIT
+maintainers         @abey79 openmaintainer
+supported_archs     arm64 x86_64
```

Comment:
does it explicitly only support these two platforms or can it build on other, by MacPorts supported, ones as well. In the latter case please just remove this line and let MacPorts set its defaults

Reply:
The reason I mentioned these archs is that glContext (and thus ModernGL) need OpenGL 3.3, which is supported by macOS 10.9 and later, which in turn restricts to the listed archs. Apart from the OGL 3.3 requirement, there is to my knowledge nothing that would preclude building on other arch, but I'm unable to test this.

I'm unsure what is the correct practice in such instance. I'm happy to remove that line if this is the right thing to do.